README.md

```diff
@@ -1,2 +1,125 @@
+      ```
+
+      <!-- TODO: Update link with our production instance of Tower -->
+   2. [Create](https://tower-dev.sagebionetworks.org/tokens) a new token in Nextflow Tower called `<stack_name>`, copy the token (which is only displayed once), and run the following command, updating `<token>` with the copied value:
+      ```
+      export NXF_TOWER_TOKEN="<token>"
+      ```
+
+   3. [Create](https://www.synapse.org/#!PersonalAccessTokens:) a new personal access token in Synapse called `<stack_name>` with all scopes enabled, copy the token (which is only displayed once), and run the following command, updating `<token>` with the copied value:
+      ```
+      export SYNAPSE_TOKEN="<token>"
+      ```
+
+   4. Run the following Docker command to configure your project in Nextflow Tower, updating `<stack_name>` with the stack name:
+      ```
+      docker run -e STACK_NAME="<stack_name>" -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN -e NXF_TOWER_TOKEN -e SYNAPSE_TOKEN -v "$HOME/.aws:/root/.aws" sagebionetworks/setup-tower-project > project-config.json
+      ```
+
+      The above Docker command performs the following tasks:
+
+      - In your AWS CLI configuration (`~/.aws/config`), it configures two profiles if they don't already exist:
+        - `sandbox` for using the `Developer` role from the `sandbox` AWS account
+        - `tower` for using the `TowerViewer` role from the `nextflow-prod` AWS account
+
+        You can login using these profiles with the following commands
+        ```
+           aws --profile sandbox sso login
+           aws --profile tower sso login
+        ```
+
+      - In Nextflow Tower, the credentials for the Forge service user were created under the name `<stack_name>`.
+
+      - In Nextflow Tower, a compute environment was created using the above credentials under the name `<stack_name> (default)`. If you need to tweak the compute environment, we recommend that you clone the default one and make adjustments as necessary.
+
+4.
+
+## AWS Accounts
+
+Two AWS accounts are managed by this repository, both of which were [bootstrapped](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2058878986/Bootstrapping+AWS+Project+Accounts) using [org-formation](https://github.com/org-formation/org-formation-cli). They are defined in this [organization.yaml](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/organization.yaml) file. The two accounts are:
+
+- `workflows-nextflow-prod` (or `nextflow-prod`) for production use
+- `workflows-nextflow-dev` (or `nextflow-dev`) for testing purposes
+
+In general, we use the `nextflow-dev` account for testing templates before deploying them into the `nextflow-prod` account. New [template configurations](#configuration) don't necessarily need to go through `nextflow-dev` first. For example, new instances of the `tower-project.yaml` template don't need to be deployed in `nextflow-dev` first.
```

Comment:
Since we're gearing the README towards Tower users, I migrated everything but the onboarding instructions to the CONTRIBUTING.md file, with a notice at the top of the README directing prospective contributors.